plugins/apitracing/src/lib/Config.cpp

```diff
@@ -0,0 +1,17 @@
+#include "Config.h"
+#include <utility>
+
+namespace ApiTracing
+{
+    Config::Config(std::filesystem::path tracingTargetsPath) : tracingTargetsPath(std::move(tracingTargetsPath)) {}
+
+    std::filesystem::path Config::getTracingTargetsPath() const
+    {
+        return tracingTargetsPath;
+    }
+
+    std::string Config::getInitialProcessName() const
```

Comment:
This function does not make sense to me. As far as I can see it will always return an empty string.

Reply:
yeah, this can be discarded